tools/docker/start.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
+set -ex
 
 cd web-platform-tests
-git pull --depth=50
+git fetch origin -n -q --depth=50
```

Comment:
Isn't the `--depth=50` a bit pointless now? Presumably we'll be updating the image once in a while, and does it even work to do a shallow fetch from within a full clone?